build/Autofac.Build.psm1

```diff
@@ -1,251 +1,261 @@
-
- .PARAMETER PackagesPath
-  Path to the "artifacts\packages" folder where packages should go.
-
- .PARAMETER VersionSuffix
-  The version suffix to use for the NuGet package version.
-#>
-function Invoke-DotNetPack
-{
-  [CmdletBinding()]
-  Param(
-    [Parameter(Mandatory=$True, ValueFromPipeline=$True, ValueFromPipelineByPropertyName=$True)]
-    [ValidateNotNull()]
-    [System.IO.DirectoryInfo[]]
-    $ProjectDirectory,
-
-    [Parameter(Mandatory=$True, ValueFromPipeline=$False)]
-    [ValidateNotNull()]
-    [System.IO.DirectoryInfo]
-    $PackagesPath,
-
-    [Parameter(Mandatory=$True, ValueFromPipeline=$False)]
-    [AllowEmptyString()]
-    [string]
-    $VersionSuffix = ""
-  )
-  Begin
-  {
-    New-Item -Path $PackagesPath -ItemType Directory -Force | Out-Null
-  }
-  Process
-  {
-    foreach($Project in $ProjectDirectory)
-    {
-      if ($VersionSuffix -eq "")
-      {
-        & dotnet build ("""" + $Project.FullName + """") --configuration Release
-      }
-      else
-      {
-        & dotnet build ("""" + $Project.FullName + """") --configuration Release --version-suffix $VersionSuffix
-      }
-      if ($LASTEXITCODE -ne 0)
-      {
-        exit 1
-      }
-
-      if ($VersionSuffix -eq "")
-      {
-        & dotnet pack ("""" + $Project.FullName + """") --configuration Release --output $PackagesPath
-      }
-      else
-      {
-        & dotnet pack ("""" + $Project.FullName + """") --configuration Release --version-suffix $VersionSuffix --output $PackagesPath
-      }
-      if ($LASTEXITCODE -ne 0)
-      {
-        exit 1
-      }
-    }
-  }
-}
-
-<#
- .Synopsis
-  Invokes dotnet test command.
-
- .Parameter ProjectDirectory
-  Path to the directory containing the project to package.
-#>
-function Invoke-Test
-{
-  [CmdletBinding()]
-  Param(
-    [Parameter(Mandatory=$True, ValueFromPipeline=$True, ValueFromPipelineByPropertyName=$True)]
-    [ValidateNotNull()]
-    [System.IO.DirectoryInfo[]]
-    $ProjectDirectory
-  )
-  Process
-  {
-    foreach($Project in $ProjectDirectory)
-    {
-      Push-Location $Project
-
-      & dotnet test --configuration Release --logger:trx
-      if ($LASTEXITCODE -ne 0)
-      {
-        Pop-Location
-        exit 3
-      }
-
-      Pop-Location
-    }
-  }
-}
-
-<#
- .SYNOPSIS
-  Restores dependencies using the dotnet utility.
-
- .PARAMETER ProjectDirectory
-  Path to the directory containing the project with dependencies to restore.
-#>
-function Restore-DependencyPackages
-{
-  [CmdletBinding()]
-  Param(
-    [Parameter(Mandatory=$True, ValueFromPipeline=$True, ValueFromPipelineByPropertyName=$True)]
-    [ValidateNotNull()]
-    [System.IO.DirectoryInfo[]]
-    $ProjectDirectory
-  )
-  Process
-  {
-    foreach($Project in $ProjectDirectory)
-    {
-      & dotnet restore ("""" + $Project.FullName + """") --no-cache
-      if($LASTEXITCODE -ne 0)
-      {
-        exit 4
-      }
-    }
-  }
-}
+# EXIT CODES

```

Comment:
Also bring in the `.psd1` change from Autofac core, which updates the module version number.